src/my-server.js

```diff
@@ -0,0 +1,23 @@
+const util = require('util');
+
+class Server {
+    constructor(app, host = '127.0.0.1', port = 3000) {
```

Comment:
IMO it's better to use constants instead of literals even for default parameter values.